rules/windows/registry/registry_event/registry_event_defender_threat_action_modified.yml

```
@@ -0,0 +1,45 @@
+title: Windows Defender Default Threat Action Modified
+id: 5a9e1b2c-8f7d-4a1e-9b3c-0f6d7e5a4b1f
```

```suggestion
id: 5a9e1b2c-8f7d-4a1e-9b3c-0f6d7e5a4b1f
related:
    - id : 1e8a9b4d-3c2a-4f9b-8d1e-7c6a5b4f3d2e
      type: similar
```
Also, please add any related rule if it’s similar but tailored to a different log source for detecting the same event.
it's applicable for other rules as well.